libraries/TGRSIAnalysis/TS3/TS3Hit.cxx

```diff
@@ -48,8 +50,8 @@ Short_t TS3Hit::GetMnemonicSegment(TFragment &frag){//could be added to TGRSIDet
 }
 
 
-TVector3 TS3Hit::GetChannelPosition(double dist) const {
-	return TS3::GetPosition(GetRing(),GetSector());
+TVector3 TS3Hit::GetChannelPosition(double offset, double dist) const {
```

Comment:
Similar suggestion here: making the arguments listed in the dist, offset order and giving offset a default value will make this much more backward compatible.